spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -1646,6 +1647,37 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde {
             None
           }
 
+        case Murmur3Hash(children, seed) if children.forall(c => supportedDataType(c.dataType)) =>
+          // TODO: support list/map/struct type for murmur3 hash
```

Comment:
This TODO seems unnecessary as other expressions also don't support nested types.